cmake/FindSIP.cmake

```diff
@@ -68,16 +68,26 @@ endif()
 
 get_filename_component(_python_binary_path ${Python3_EXECUTABLE} DIRECTORY)
 
-find_program(SIP_EXECUTABLE sip
+find_program(SIP_EXECUTABLE sip5
     HINTS ${CMAKE_PREFIX_PATH}/bin ${CMAKE_INSTALL_PATH}/bin ${_python_binary_path} ${Python3_SITELIB}/PyQt5
 )
 
+execute_process(
+  COMMAND sip-module --sip-h --target-dir ${CMAKE_CURRENT_BINARY_DIR} PyQt5.sip
```

Comment:
The module name should be an CMAKE_OPTION, defaulting to 'sip' for legacy compat.

Reply:
Moreover, you should not rebuild the existing PyQt5.sip module and conflict with an already installed one. If you really want to build your own (why?), use a unique name.